### ISSUE TITLE
Site Performance and SEO

### ISSUE THREAD

**Issue by drod0942** [lilyicecream/Hardcore-ratings#7]
Using Google’s PageSpeed Insights (https://pagespeed.web.dev/), your site should have 90+ values for all criteria – Performance, Accessibility, Best Practices, and SEO.
